financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerViewModel.kt

```diff
@@ -27,18 +32,43 @@ internal class AccountPickerViewModel @Inject constructor(
 ) : MavericksViewModel<AccountPickerState>(initialState) {
 
     init {
+        viewModelScope.launch {
+            val manifest = getManifest()
+            setState {
+                copy(
+                    accessibleDataCalloutModel = AccessibleDataCalloutModel(
+                        businessName = ConsentTextBuilder.getBusinessName(manifest),
+                        permissions = manifest.permissions,
+                        isStripeDirect = manifest.isStripeDirect ?: false,
+                        dataPolicyUrl = ConsentUrlBuilder.getDataPolicyUrl(manifest)
+                    )
+                )
+            }
+        }
         suspend {
-            val authSession = requireNotNull(getManifest().activeAuthSession)
-            pollAuthorizationSessionAccounts(authSession.id)
+            val manifest = getManifest()
+            val authSession = requireNotNull(manifest.activeAuthSession)
+            val partnerAccountList = pollAuthorizationSessionAccounts(authSession.id)
+            partnerAccountList.data.map { account ->
+                AccountPickerState.PartnerAccountUI(
+                    account = account,
+                    enabled = manifest.paymentMethodType == null ||
+                        account.supportedPaymentMethodTypes.contains(manifest.paymentMethodType)
+                )
```

Comment:
@kgaidis-stripe this is how web currently decides wether or not the account is selectable on the picker : )